appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/RPCFailedStatusException.java

```diff
@@ -0,0 +1,16 @@
+package com.google.apphosting.vmruntime;
+
+import com.google.apphosting.api.ApiProxy;
+
+public class RPCFailedStatusException extends ApiProxy.RPCFailedException {
+  private final int statusCode;
+  public RPCFailedStatusException(String packageName, String methodName, int statusCode)
+  {
```

Comment:
Style guide: we usually follow the Google style https://google.github.io/styleguide/javaguide.html